pcdsdevices/valve.py

```diff
@@ -227,6 +227,34 @@ class VGC(VRC):
                     doc=('individual valve MPS state for debugging'))
 
 
+class VFS(Device):
+    """Class for Fast Shutter Valve."""
+    request_close = Cpt(EpicsSignalWithRBV, ':CLS_SW', kind='normal',
+                        doc='Request Fast Shutter to Close')
+    request_open = Cpt(EpicsSignalWithRBV, ':OPEN_SW', kind='normal',
+                       doc='Request Fast Shutter to Open')
+    reset_vacuum_fault = Cpt(EpicsSignalWithRBV, ':FAULT_RESET', kind='normal',
+                             doc=('Reset Fast Shutter Vacuum Faults: fast'
+                                  'sensor triggered, fast sensor turned off'))
+    override_status = Cpt(EpicsSignalWithRBV, ':VFS_OVRD_ON', kind='normal',
```

Comment:
If it is a readback shouldn’t it be an EpicsSignalRO?

Reply:
For Reset_vacuum Fault and override status, they are both settable values. Override status, is some overarching override, should the name get changed?

Reply:
probably name and docstrings.

Reply:
There's just a mismatch between `EpicsSignalWithRBV` and `doc=Epics Readback on Override mode`. Perhaps the correct thing to do is fix the docstring if this is a settable?

Reply:
My suggestion for a rename if you are considering one is `override_mode`

Reply:
Thank you Zach, `override_mode` is much better. Doc strings have been updated